provisioning_client/devdoc/azure_provisioning_faq.md

```diff
@@ -0,0 +1,39 @@
+# Azure Iot Provisioning Client FAQ
+
+Here is where we shall try to answer question pertaining to the IoT Provisioning Client.  Please read the following documents for an overview of the DPS service.
+
```

Comment:
I'd use "Device Provisioning Service" and link to our docs: https://docs.microsoft.com/en-us/azure/iot-dps/